### PR TITLE
Docs Pull Request - 260123

### PR DESCRIPTION
**Pull request by miquelramirez**
Hello, 

this PR fixes a typo and suggests a code example for the PDDL reader parser/factory that provides more handholding. I have also ported a piece of code in Python 2.x to Python 3.1x to use the newest string formatting capabilities in the notebook for the Temporal Planning example.

Thank you and congratulations for making all the way to 1.0.0.

### CODE REVIEW

File: docs/interoperability.rst [hunk 1]
```diff
@@ -6,11 +6,18 @@ In order to make the library immediately usable for people having existing code
...
     :caption: PDDLReader example
-    :lines: 12-16
+
+        from unified_planning.io import PDDLReader
+        # ...
+        reader = PDDLReader()
+        # Reader used with both domain and problem file
+        # domain_filename = ... # path of the PDDL domain file
+        # problem_filename = ... # path of the PDDL problem file
+        problem = reader.parse_problem(domain_filename, problem_filename)
```
Comment (alvalentini) [inline docs/interoperability.rst:20]: To include the `import` you can update the previous code as follows:

```
.. literalinclude:: ./code_snippets/pddl_interop.py
    :caption: PDDLReader example
    :lines: 2,11-16
```

**Review by alvalentini — CHANGES_REQUESTED** [review]
Thanks for the PR! I added a comment to suggest you a different way to achieve the same result.

File: docs/interoperability.rst (hunk 1 above)
Comment (miquelramirez) [reply docs/interoperability.rst]: Suggested change has been commmitted to the branch, cheers.

File: docs/interoperability.rst [hunk 2]
```diff
@@ -6,11 +6,11 @@ In order to make the library immediately usable for people having existing code
...
+The library offers a comprehensive and well-tested parser for PDDL 2.1 level 3 (i.e. up to temporal planning without continuous change). The parser can be invoked as shown in the example below and can be used either to parse a whole planning instance (i.e. a domain and a problem), or just to parse the domain file, resulting in an incomplete ``Problem`` object that can be completed using the UP API as shown below. This latter use-case allows to model the action schemata using PDDL and then formulate the problem(s) using the UP (possibly using data available at runtime for doing so).
 
-.. literalinclude:: ./code_snippets/pddl_interop.py
+.. literalinclude::
```
Comment (alvalentini) [inline docs/interoperability.rst:11]: You should undo this change:

```suggestion
.. literalinclude:: ./code_snippets/pddl_interop.py
```

**Review by alvalentini — APPROVED** [review]
